packages/sdk/src/dataProtector/IExecDataProtector.ts

```diff
@@ -50,13 +51,19 @@ import {
   RevokedAccess,
   SetProtectedDataToSubscriptionParams,
   SetProtectedDataToSubscriptionResponse,
-  SetSubscriptionOptionsParams,
-  SetSubscriptionOptionsResponse,
+  SetSubscriptionParams,
+  SetSubscriptionParamsResponse,
   SubscribeParams,
   Taskid,
   TransferParams,
   TransferResponse,
   Web3SignerProvider,
+  GetRentersParams,
+  Renters,
+  AddToCollectionResponse,
+  Creator,
+  GetSubscribersResponse,
+  SubscribeResponse,
```

Comment:
Can we keep an alphabetical order in this list of imported members?

Reply:
Yep. I thought it was made automatically, the role of the config prettier to do that